<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiSelect</class>
 <widget class="QDialog" name="DiSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>348</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>771</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>DI</string>
   </property>
   <widget class="QPushButton" name="none">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>156</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>사용안함</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="di01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>36</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DI 1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="di02">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>36</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DI 2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="di03">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>36</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DI 3</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="di04">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>36</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DI 4</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="di06">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>36</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DI 6</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="di05">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>36</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DI 5</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>121</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>121</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>DiSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
